Kernel/Process.h

```diff
@@ -661,8 +665,7 @@ inline void Process::for_each_thread(Callback callback) const
     if (my_pid == 0) {
         // NOTE: Special case the colonel process, since its main thread is not in the global thread table.
         Processor::for_each(
-            [&](Processor& proc) -> IterationDecision
-            {
+            [&](Processor& proc) -> IterationDecision {
```

Comment:
Should I revert these odd clang-format changes? :/ I'm using clang-format-9...

Reply:
Yes, please use clang-format 10 :)

Reply:
Welp, everything left in the clang-format department is left from using clang-format-10